cmd/gen-go-sample/main.go

```diff
@@ -332,16 +332,21 @@ func (g *generator) genSample(ifaceName string, methConf GAPICMethod, regTag str
 	var argStr string
 	if len(argNames) > 0 {
 		var sb strings.Builder
-		for i, n := range argNames {
-			if e, ok := argTrees[i].typ.desc.(*descriptor.EnumDescriptorProto); ok {
-				en, err := enumType(g.descInfo, e, g)
+		for i, name := range argNames {
+			var typ string
```

Comment:
We don't need to declare `typ` like this if it's being redeclared immediately via `typ, ok :=`.

Reply:
Ah my bad!